src/DomainServer.ts

```diff
@@ -387,19 +378,17 @@ class DomainServer {
     #nodeKilled = (node: Node): void => {
         // C++  void Application:: nodeKilled(Node* node)
 
-        // WEBRTC TODO: Address further code - for AssetServer node.
-
         const nodeType = node.getType();
-        if (nodeType === NodeType.AudioMixer) {
-            console.warn("DomainServer: AudioMixer support not implemented!");
 
-            // WEBRTC TODO: Address further code - for AssetServer node.
+        // AudioMixer node is handled in AudioMixer.ts.
 
-        } else if (nodeType === NodeType.EntityServer) {
+        if (nodeType === NodeType.EntityServer) {
```

Comment:
This series of if-else should probably become a switch statement for readability as it gets longer

Reply:
Ditto, C++ has 3, TypeScript currently has 2 which will likely reduce in time.